application/jaxrs-redis-quarkus/src/test/java/webapp/tier/service/RedisServiceTest.java

```diff
@@ -44,7 +44,7 @@ public static void after() {
 		server = null;
 	}
 
-	private Jedis createJedisMock() {
+	static Jedis createJedisMock() {
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=5948487)